internal/config/config.go

```diff
@@ -44,7 +44,8 @@ type Config struct {
 	MaxRecordAge       time.Duration
 	EnableProviders    bool
 	EnableValues       bool
-	ProvidersOptions   []providers.Option
+	ProvidersOptions   []providers.Option //XXX: get rid?
```

Comment:
I think it's fine to drop this as long as we make this a minor release and explain how to do this in the release notes for this library (i.e. create a ProviderManager with the relevant options and pass that in as a ProviderStore)